messages/php/src/Id/FallbackUuidGenerator.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cucumber\Messages\Id;
+
+final class FallbackUuidGenerator implements IdGenerator
+{
+    /**
+     * Inspired by
+     * @see https://github.com/clue/reactphp-clickhouse
+     */
+    public function newId(): string
+    {
+        $bytes = random_bytes(16);
+
+        /**
+         * set MSBs of octet 6 to 0100... to indicate v4
+         *
+         * @see https://datatracker.ietf.org/doc/html/rfc4122#section-4.1.3
+         */
+        $bytes[6] = chr(ord($bytes[6]) & 0b0000_1111 | 0b0100_0000);
```

Comment:
This `chr` and `ord` looks expensive. I think that with two random longs you can achieve the same.

Reply:
Nevermind, PHP doesn't have well defined numerical sizes.

I imagine there is a faster way but I don't know enough to suggest one.

Reply:
Yes you can inspect PHP_INT_SIZE (and it'll be 32 or 64 unless it's some crazy exotic platform) but that complicates the algo

there are BigInt extensions that you can use on a project, but for a library you can't really require too many extensions as they aren't pulled in by package management

I'll re-address this part of the code and see if it can be simplified

Reply:
I've been doing PHP for over 20 years and found out today you can use bitwise operators directly on strings 🤯

Dynamic typing rocks

I've reworked this bit but GH has lost the ref because the filename changed